cache/sampleStore.js

```diff
@@ -0,0 +1,252 @@
+/**
+ * Copyright (c) 2017, salesforce.com, inc.
+ * All rights reserved.
+ * Licensed under the BSD 3-Clause license.
+ * For full license text, see LICENSE.txt file in the repo root or
+ * https://opensource.org/licenses/BSD-3-Clause
+ */
+
+/**
+ * ./cache/sampleStore.js
+ *
+ * Redis Cache
+ */
+'use strict'; // eslint-disable-line strict
+const redisClient = require('./redisCache').client.sampleStore;
+const Sample = require('../db').Sample;
+const featureToggles = require('feature-toggles');
+const PFX = 'samsto';
+const SEP = ':';
+const constants = {
+  featureName: 'enableRedisSampleStore',
+  fieldsToStringify: {
+    aspect: [
+      'relatedLinks',
+      'tags',
+      'criticalRange',
+      'warningRange',
+      'infoRange',
+      'okRange',
+    ],
+    sample: ['relatedLinks'],
+  },
+  indexKey: {
+    aspect: PFX + SEP + 'aspects',
+    sample: PFX + SEP + 'samples',
+    subject: PFX + SEP + 'subjects',
+  },
+  objectType: { aspect: 'aspect', sample: 'sample', subject: 'subject' },
+  prefix: PFX,
+  separator: SEP,
```

Comment:
Nevermind, I just thought we had sequelize commands defined in constants file, like $iLike etc, so we can define these strings too, but not a big deal.